packages/venia-ui/lib/components/RichContent/PageBuilder/ContentTypes/Column/configAggregator.js

```diff
@@ -1,21 +1,25 @@
 import {
     getAdvanced,
     getBackgroundImages,
+    getBorder,
+    getCssClasses,
+    getMargin,
+    getTextAlign,
     getVerticalAlignment
 } from '../../utils';
 
 export default node => {
     return {
         minHeight: node.style.minHeight ? node.style.minHeight : null,
-        textAlign: node.style.textAlign,
         display: node.style.display,
         width: node.style.width,
-        ...getVerticalAlignment(node),
         backgroundColor: node.style.backgroundColor,
+        ...getAdvanced(node),
         ...getBackgroundImages(node),
-        cssClasses: node.getAttribute('class')
-            ? node.getAttribute('class')
-            : null,
-        ...getAdvanced(node)
+        ...getBorder(node),
```

Comment:
`getAdvanced` internally calls `getBorder`, `getCssClasses` etc. So you only need `getAdvanced` or the others.

Please see: https://github.com/magento/pwa-studio/blob/5b50350c31fa5607d5d4e61db39aab83b32d30c6/packages/venia-ui/lib/components/RichContent/PageBuilder/utils.js#L84-L92